fideslib/core/config.py

```diff
@@ -0,0 +1,272 @@
+import hashlib
+import logging
+import os
+from pathlib import Path
+from typing import Any, Dict, List, MutableMapping, Optional, Tuple, Type, Union
+
+import bcrypt
+import tomli
+from pydantic import AnyHttpUrl, BaseSettings, PostgresDsn, ValidationError, validator
+from pydantic.env_settings import SettingsSourceCallable
+
+from fideslib.exceptions import MissingConfig
+from fideslib.logging.logger import NotPii
+
+logger = logging.getLogger(__name__)
+
+
+class FidesSettings(BaseSettings):
+    """Class used as a base model for configuration subsections."""
+
+    class Config:  # pylint: disable=C0115
```

Comment:
Commented this way at the end of the line I believe it only disables for the single line, though I could be wrong about that. At lease that is the way it works in flake8 which is what I am used to. It also works this way in mypy.

That said I'm open to however everyone wants to do it. If we are going to disable at a broader level my vote would be in `pyproject.toml` rather than adding to the top of every file.

Reply:
The main point was just that we should be deliberate about where we place them. It's a nuisance to maintain lots of single-line disables in the same file, but if it's the _only_ file where that particular rule is disabled, it doesn't make sense to disable the rule globally in the `pyproject.toml`. I also strongly prefer the long-form over the pylint code, for readability.